modules/generate-diagnostic-bundle-using-roxctl-cli.adoc

```diff
@@ -10,10 +10,31 @@ You can generate a diagnostic bundle by using the `roxctl` CLI.
 .Prerequisites
 * To generate a diagnostic bundle, you need `read` permission for the `DebugLogs` resource.
 
+* You have configured the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Set the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables:
```

Comment:
Instead of using `ROX_API_TOKEN`, this is also possible with `-p "$ROX_PASSWORD"`

Reply:
@rukletsov IIUC, all instances of `ROX_API_TOKEN` can be replaced with `ROX_PASSWORD` and the export command would be:
`$ export ROX_PASSWORD=_<rox_password>_` instead of `$ export ROX_API_TOKEN=_<api_token>_`

Is that correct?

Reply:
example with password https://github.com/stackrox/stackrox/blob/e2c428cfdb8d7d3d542fb8e247962e0402a8cb32/scripts/ci/lib.sh#L159

It's worth to mention that password is admin password

Reply:
Done.